packages/preview/typxidian/0.1.1/src/dependencies.typ

```diff
@@ -1,8 +1,10 @@
 #import "@preview/cetz:0.4.2" // drawing inspired by tikz
+#import "@preview/cetz-plot:0.1.2"
+#import "@preview/plotsy-3d:0.2.0"
```

Comment:
While you are at it, it might also be worth it to update to 0.2.1, which fixes a few bugs.